src/lib/components/Text/Text.js

```diff
@@ -7,12 +7,56 @@ import styles from './Text.module.scss';
 
 import { getColor } from '../Treebank/config';
 
-// eslint-disable-next-line react/prop-types
-const wordToSpan = (config, active, toggleActive, word) => {
-  const { $: { id, form, postag } } = word;
+const formatId = (id) => id.padStart(4, '0');
```

Comment:
Thanks to the new [padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart) function, [this](https://github.com/alpheios-project/arethusa/blob/master/app/js/arethusa.util/helpers.js#L5) whole function can be reduced to a single line!